Dockerfile

```diff
@@ -14,7 +14,8 @@ WORKDIR /app
 
 RUN apk add --update nodejs npm
 RUN apk add --no-cache tini
+RUN apk add --no-cache bash
 
 ENTRYPOINT ["/sbin/tini", "--"]
 
-CMD ["node", "index.js"]
\ No newline at end of file
+CMD ["./wait-for-it.sh" , "mongodb:27017" , "--strict" , "--timeout=30" , "--" , "node", "index.js"]
```

Comment:
Bu degisiklik sadece local'de calisirken ve docker-compose kullanirken karsilasacagimiz bir sorun degil mi? Eger oyleyse, wait for it eklemektense (ki production'da mongodb:27017 datasi olmayacak), docker-compose'a restart on failure eklemeyi oneririm

Reply:
sadece local ortamda calisacaksa evet buna pek gerek yok, `restart: always` dahi is gorecektir. Geri almak ve restart_policy icin commit atacagim.